README.md

```diff
@@ -11,58 +10,116 @@
+Bit `add` allows to track a subset of files or directories as a reusable code component. Classic use cases would be web components (native, react, angular, etc.), utility functions or any other nodejs module.
 
-- **Component discovery engine.** Bit comes with an integrated search engine that uses expressive linguistic models to make your components discoverable even when you forget the exact name you gave each component. This also helps for collaborating as a team on shared Scopes.
+```sh
+bit add src/utils/left-pad.js
+# Tracked utils/left pad with files 
+```
 
-## Documentation
+You can use glob patterns to track a vast amount of components at once:
 
-[Docs](https://teambit.github.io/bit)
+```sh
+bit add src/utils/*.js
+# Tracked 24 new components
+```
 
-[Bit Scope](https://teambit.github.io/bit/bit-scope.html)
+### Commit
 
-[Bit component](https://teambit.github.io/bit/bit-component.html)
+Bit `commit`, commits changes to new and existing components in your application.
 
-[Bit environment](https://teambit.github.io/bit/bit-component.html#component-environment)
+To check which components were changed or added and about to be committed, you can use `bit status`:
 
-[Bit on the server](https://teambit.github.io/bit/bit-on-the-server.html)
+```sh
+bit status
+# New components:
+#   utils/pad-left
+# Modified components:
+#   utils/is-string
+```
 
-[CLI reference](https://teambit.github.io/bit/cli-reference.html)
+To commit all changes use:
 
-## Installation
+```sh
+bit commit -am ‘committed my first tracked code components'
+```
 
-For our different installation methods, please visit our docs [installation section](https://teambit.github.io/bit/installation.html).
+Now all your components are staged, and ready to be pushed or used from any other project.
 
-## Quick start
+### Export
+
+You can push staged/committed components to any remote Scope hosted on [bitsrc.io](https://bitsrc.io) or created on any machine (and connected via SSH).
+
+You can set up a free Bit Scope at Bit’s [community hub](https://bitsrc.io), and follow the [setup instructions](https://docs.bitsrc.io/en/article/07-create-a-free-bitsrc-scope).
+
+It’s also possible to easily set up a [remote scope on your own
+computer](https://teambit.github.io/bit/getting-started.html#setup-a-remote-scope).
+
+Once you have a remote scope ready, run the export command:
+
+```sh
+bit export bit.utils
+# bit.utils is your Scope name
+```
+
+### Import
+
+Bit `import` enables to instal component (as an application part) you’ve tracked in any destination on your project’s file system.
+
+Let's import the component we just created to a new project.
+
+1. Create a new project.
+2. Initialize a new scope using the bit init command.
+3. Import the component
+  ```sh
+  bit import my-scope/left-pad
+  ```
+
+The component is now it in the components directory, ready to be used in your code.
+
+**Use:**
+
+```js
+const component = require(./left-pad);
```

Comment:
@itaymendel The require will be something like:
```js
require('./components/utils/left-pad')
```
Also add the quotes to the path